openforcefield/tests/test_parameters.py

```diff
@@ -2050,15 +2050,25 @@ def test_charge_increment_type_wrong_num_increments(self):
                 charge_increment3=-0.1 * unit.elementary_charge,
             )
 
-        with pytest.raises(
-            SMIRNOFFSpecError,
-            match="initialized with unequal number of tagged atoms and charge increments",
-        ) as excinfo:
-            ci_type = ChargeIncrementModelHandler.ChargeIncrementType(
-                smirks="[#6:1]-[#7:2]-[#6]",
-                charge_increment1=0.05 * unit.elementary_charge,
-            )
+        ci_type = ChargeIncrementModelHandler.ChargeIncrementType(
+            smirks="[#6:1]-[#7:2]-[#6]",
+            charge_increment1=0.05 * unit.elementary_charge,
+        )
 
+    def test_charge_increment_one_ci_missing(self):
```

Comment:
I agree with your reasoning behind making these models different _pre_-parametrization, but I would like to see a test that ensures their behavior is identical or sufficiently similar. My reading of `test_charge_increment_model_invalid_number_of_cis` is that the two cases are tested only to the extend that they don't error out. `test_charge_increment_model_one_less_ci_than_tagged_atom` probably covers enough behavior to practically safeguard ourselves, but it would be nice to have a test that runs these two `ChargeIncrementType`s through parametrizationa nd ensures they produced the same result.

Reply:
Good call. Added in bcc27be